merge.go

```diff
@@ -81,7 +90,13 @@ func (m *MergeSetConfig) ToString() (string, error) {
 	if m.Target != nil && (reflect.TypeOf(m.Target) == reflect.TypeOf(ParamString(""))) {
 		sb.WriteString(m.Name)
 		sb.WriteRune(' ')
-		sb.WriteString(EqualToOperator.String())
+
+		if m.Type == MERGE {
+			sb.WriteString(PlusEqualOperator.String())
+		} else {
+			sb.WriteString(EqualToOperator.String())
+		}
+
```

Comment:
Add validation of `Operator` above:
```
	operator := m.Operator
	if operator == "" {
		operator = EqualToOperator
	} else if operator != EqualToOperator && operator != PlusEqualOperator {
		return "", errors.New("invalid SET operator")
	}
```

Then just use it here:
```suggestion
		sb.WriteString(operator.String())
```

Reply:
80b7e27b9dea70057300e04368187c6b80c3dce3